src/Entity/Conference.php

```diff
@@ -60,11 +60,6 @@ class Conference
      */
     private $description;
 
-    /**
-     * @ORM\Column(name="location", type="string", length=255)
-     */
-    private $location;
-
```

Comment:
Be careful here, we will lost data stored in this field. Be sure that your migration will move data from Location to City or Country fields